### ISSUE TITLE
Cant connect to plexconnect

### ISSUE THREAD

**Issue by wouter-asterisk** [iBaa/PlexConnect#448]
File "C:\Users\Wouter\Downloads\PlexConnect-master\PlexAPI.py", line 432, in getXMLFromPMS
    response = urllib2.urlopen(request, timeout=20)
  File "C:\Python27\lib\urllib2.py", line 154, in urlopen
    return opener.open(url, data, timeout)
  File "C:\Python27\lib\urllib2.py", line 435, in open
    response = meth(req, response)
  File "C:\Python27\lib\urllib2.py", line 548, in http_response
    'http', request, response, code, msg, hdrs)
  File "C:\Python27\lib\urllib2.py", line 473, in error
    return self._call_chain(_args)
  File "C:\Python27\lib\urllib2.py", line 407, in _call_chain
    result = func(_args)
  File "C:\Python27\lib\urllib2.py", line 556, in http_error_default
    raise HTTPError(req.get_full_url(), code, msg, hdrs, fp)
HTTPError: HTTP Error 401: Unauthorized

Can someone help me, cause i dont know where 2 start.

**Comment by roidy** [comment]
>HTTPError: HTTP Error 401: Unauthorized

Sign into both your PMS and the PlexConnect app.

**Comment by wouter-asterisk** [comment]
@roidy 
This is still what i get:
15:03:23 PlexAPI: PlexGDM - looking up Plex Media Server
15:03:23 PlexAPI: ***
15:03:24 PlexAPI: GDM: Servers discovered: 1
15:03:24 PlexAPI: Servers (local, plex.tv, MyPlex): 2
15:03:24 WebServer: serving .xml: /PlexConnect.xml
15:03:24 WebServer: serving /js/utils.js
15:03:24 WebServer: serving /js/handlePlexConnectNavbar.js
15:03:24 WebServer: serving .xml: /library/sections
15:03:24 PlexAPI: No Response from Plex Media Server
15:03:24 PlexAPI: We failed to reach a server. Reason: Unauthorized
15:03:24 PlexAPI: Traceback:
Traceback (most recent call last):
  File "C:\Users\Wouter\Downloads\PlexConnect-master\PlexAPI.py", line 432, in getXMLFromPMS
    response = urllib2.urlopen(request, timeout=20)
  File "C:\Python27\lib\urllib2.py", line 154, in urlopen
    return opener.open(url, data, timeout)
  File "C:\Python27\lib\urllib2.py", line 435, in open
    response = meth(req, response)
  File "C:\Python27\lib\urllib2.py", line 548, in http_response
    'http', request, response, code, msg, hdrs)
  File "C:\Python27\lib\urllib2.py", line 473, in error
    return self._call_chain(*args)
  File "C:\Python27\lib\urllib2.py", line 407, in _call_chain
    result = func(*args)
  File "C:\Python27\lib\urllib2.py", line 556, in http_error_default
    raise HTTPError(req.get_full_url(), code, msg, hdrs, fp)
HTTPError: HTTP Error 401: Unauthorized

15:03:24 WebServer: serving /js/utils.js
15:03:24 WebServer: serving /js/sectionHoldSelect.js

(i have my pms on and open in browser and my plexconnect.py is saying this)

**Comment by CyberGhost84** [comment]
https://github.com/iBaa/PlexConnect/issues/447#issuecomment-247818004

**Comment by Fetchered** [comment]
Had a similar problem before and after re-JB'ing my ATV2. I managed to find a solution, but for a backstory (and to ensure others looking for a solution see the exact problem which needed this exact solution) here are the errors I was getting:

5:25:25 PlexAPI: No Response from Plex Media Server
15:25:25 PlexAPI: We failed to reach a server. Reason: [Errno 8] nodename nor servname provided, or not known
15:25:25 PlexAPI: Traceback:
Traceback (most recent call last):
  File "/private/var/stash/Applications/PlexConnect/PlexAPI.py", line 432, in getXMLFromPMS
    response = urllib2.urlopen(request, timeout=20)
  File "/usr/lib/python2.7/urllib2.py", line 127, in urlopen
    return _opener.open(url, data, timeout)
  File "/usr/lib/python2.7/urllib2.py", line 404, in open
    response = self._open(req, data)
  File "/usr/lib/python2.7/urllib2.py", line 422, in _open
    '_open', req)
  File "/usr/lib/python2.7/urllib2.py", line 382, in _call_chain
    result = func(*args)
  File "/usr/lib/python2.7/urllib2.py", line 1222, in https_open
    return self.do_open(httplib.HTTPSConnection, req)
  File "/usr/lib/python2.7/urllib2.py", line 1184, in do_open
    raise URLError(err)
URLError: <urlopen error [Errno 8] nodename nor servname provided, or not known>

Now at this point, my ATV 2 is on the same network, and without signing in I could see under Discovery that I can see my Plex server, but there were no selections available under any of the headings. So I read #447 and saw the 'log out/pass change/log in' comment, and someone else mentioning adding an IP to the 'List of IP addresses and networks that are allowed without auth' ( #445 ), so I logged in (I wasn't logged in at this point) and got the 'HTTPError: HTTP Error 401: Unauthorized' error with no Discovered PMS. So I logged out, saw my discovered PMS again, and decided to add my IP address to the aforementioned list, closed PlexConnect, then re-opened and voila! I now have access to my PMS. 

TL;DR: Log in to your Plex account in PlexConnect, then log out. Add your IP address to the 'List of authorized IP addresses', close PlexConnect and re-open. This is by no means a permanent fix, but one to tide you over until the real issue (whatever that may be) is resolved.

**Comment by haxcop** [comment]
Try using plexconnect into the \program files\ this should work

**Comment by moodyblue** [comment]
Help requests should be posted in the forum

https://forums.plex.tv/categories/appletv-plexconnect
